rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -142,6 +142,26 @@ rosidl_write_generator_arguments(
 file(MAKE_DIRECTORY "${_output_path}")
 file(WRITE "${_output_path}/__init__.py" "")
 
+if(NOT "${_generated_msg_py_files} " STREQUAL " ")
+  list(GET _generated_msg_py_files 0 _msg_file)
+  get_filename_component(_msg_package_dir1 "${_msg_file}" DIRECTORY)
+  get_filename_component(_msg_package_dir2 "${_msg_package_dir1}" NAME)
+endif()
+
+if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
+  ament_python_install_module("${_output_path}/__init__.py"
+    DESTINATION "${PROJECT_NAME}"
+  )
+
+  ament_python_install_module("${_output_path}/__init__.py"
+    DESTINATION "${PROJECT_NAME}/${_msg_package_dir2}"
+  )
+
+  install(FILES ${_generated_msg_py_files}
+    DESTINATION "${PYTHON_INSTALL_DIR}/${PROJECT_NAME}/${_msg_package_dir2}"
```

Comment:
Yes, but unfortunately when I replaced this with `ament_python_install_module`, the code that checks if the file to be installed exists fails. Could this be because that doesn't run at install time?